bitnami/odoo/Chart.yaml

```
@@ -27,4 +27,4 @@ name: odoo
 sources:
   - https://github.com/bitnami/bitnami-docker-odoo
   - https://www.odoo.com/
-version: 21.0.1
+version: 21.0.2
```

The change is including a minor feature, is not fixing a bug
```suggestion
version: 21.1.0
```